app/lib/models.py

```diff
@@ -0,0 +1,11 @@
+from app import db
+
+
+class Base(db.Model):
```

Comment:
Also, instead of calling this `Base` lets call this `SomethingMixin`. Totally my bad earlier, but after looking at the documentation for sqlalchemy a bit more closely, this seems like the better way to do things: http://docs.sqlalchemy.org/en/rel_1_0/orm/extensions/declarative/mixins.html

Hence we need to change several things:
1) Out app's `__init__.py` file should perhaps make a variable called Base that makes the declarative base. https://pythonhosted.org/Flask-SQLAlchemy/api.html#flask.ext.sqlalchemy.SQLAlchemy.make_declarative_base
2) This class should be a "Mixin"
3) Whenever we want to use our Mixin, we should declare a class with both `Base` and `SomethingMixin`. It looks like the ordering doesn't matter, but lets be consistent.